feat(page-header-standardised): Adding component - FRONT-1802 (#1782)
ec-europa/europa-component-library: src/implementations/twig/components/page-header-standardised/README.md

# ECL Page Header Standardised component

npm package: `@ecl/twig-component-page-header-standardised`

```shell
npm install --save @ecl/twig-component-page-header-standardised
```

### Parameters

- **"title"** (string) (default: '') Title of header
- **"description"** (string) (default: '') Description of header
- **"meta"** (string) (default: '') Meta of header

> - **"breadcrumb"** (associative array) (default: '') Predefined structure compatible with EC Breadcrumb

Reviewer: ```suggestion
- **"breadcrumb"** (associative array) (default: '') Predefined structure for the ECL Breadcrumb
```

- **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
- **"extra_attributes"** (optional) (array) (default: []) Extra attributes
  - "name" (string) Attribute name, eg. 'data-test'
  - "value" (string) Attribute value, eg: 'data-test-1'

> - **"\_compliance**"\_ (boolean) (default: false) Activates debug

Reviewer: ```suggestion
- **"\_compliance\_**" (boolean) (default: false) Activates debug
```
